go/mysql/flavor.go

```diff
@@ -62,6 +62,9 @@ type flavor interface {
 	// stopSlave returns the command to stop the slave.
 	stopSlaveCommand() string
 
+	// sendBinlogFileDumpCommand sends the packet required to start streaming from file:post
```

Comment:
No need to separate out the flavors. The cool thing is that the same command will work for both mariadb and mysql. The two versions diverged only due to GTID.